cmd/groundstation/main.go

```diff
@@ -10,29 +10,28 @@ import (
 )
 
 func main() {
-	slog.Info("Starting groundstation")
```

Comment:
NIT: Theirs no reason to remove it. I'd like to have it log immediatly

Reply:
Ah, I thought this message was indicating that the satellite->groundstation server was about to be started, so I moved it down to just before that actually occured, like how there's one for the api server

Reply:
No, it's that the *binary* has started. Their can be a seperate line for the server

Reply:
Cool, will add back